libs/datamodel/core/tests/reformat/reformat_implicit_relations.rs

```diff
@@ -563,7 +563,7 @@ fn must_add_back_relation_fields_for_self_relations() {
           id    Int     @id
           sonId Int?
           son   Human?  @relation(fields: [sonId], references: [id])
-          Human Human[] @relation("HumanToHuman")
+          Human Human[]
```

Comment:
I can elaborate on why these changes are correct, or at least not worse than before, I went through these.